crates/ott-collector/src/main.rs

```diff
@@ -1,10 +1,156 @@
 #[macro_use]
 extern crate rocket;
 
+use rocket::serde::{json::Json, Serialize};
+
+#[derive(Serialize)]
+#[serde(crate = "rocket::serde")]
+struct SystemState {
+    balancers: Vec<Balancer>,
+}
```

Comment:
This is not the same shape as the `SystemState` type in `ott-vis-common`. This serializes into an object instead of an array.

Reply:
didn't see this before committing